src/tap/include/tap/MakeTapcell.h

```diff
@@ -33,7 +33,6 @@
 //
 ///////////////////////////////////////////////////////////////////////////////
 
-
 #ifndef MAKE_TAPCELL_H
 #define MAKE_TAPCELL_H
```

Comment:
Use `#pragma once` instead of `#ifndef MAKE_TAPCELL_H ` and `#define MAKE_TAPCELL_H `